### ISSUE TITLE
spellchecking: text scrunched up when base font size increased to 32px

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#224]
Looks like the margins are missing. Makes it harder to scan, but technically this probably isn't a fail.

**Comment by alrra** [comment]
Closing this in favor of #257.
